.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: commcare-export tests
+on:
+  pull_request:
+    branches:
+      - master
+env:
+  DB_USER: db_user
+  DB_PASSWORD: Password123
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    services:
+      mssql:
+        image: mcr.microsoft.com/mssql/server:2017-latest
+        env:
+          SA_PASSWORD: ${{ env.DB_PASSWORD }}
+          ACCEPT_EULA: 'Y'
+        ports:
+          - 1433:1433
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: ${{ env.DB_PASSWORD }}
+          POSTGRES_USER: ${{ env.DB_USER }}
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, '3.10'] # Version 3.6 with arch x64 not found
```

Comment:
Added 3.9 and 3.10. Should we drop more than 3.6?

Reply:
I think we should keep it until it's past EOL (2023-06-27).